Neos.NodeTypes/Configuration/NodeTypes.Content.Headline.yaml

```diff
@@ -6,3 +6,6 @@
     label: i18n
     icon: 'icon-header'
     position: 100
+  options:
+    fusion:
+      prototypeGenerator: Neos\Neos\Domain\Service\DefaultContentPrototypeGenerator
```

Comment:
@mficzel: what do you think about adding the the prototype code directly to fusion files? It feels strange to still use the deprecated class internally.

Next steps could be to rewrite the Fluid to Fusion. If thats a good idea, that could be good-first-issues

Reply:
@daniellienert I added the prototype code to the fusion of the subpackages we like and are maintaining actively. 

For Neos.NodeTypes itself i took the minimal needed effort as we are saying for a while that Neos.NodeTypes should not be used anymore. It could be done and probably would be god but i would see it in a follow up pr.